### ISSUE TITLE
Can the avg loss by iteration be saved?

### ISSUE THREAD

**Issue by Eve66666** [AlexeyAB/darknet#5539]
Can the data points (x=iteration,y=avg loss) used to draw 'chart.png' in the training process be saved as TXT file?

**Comment by mfaramarzi** [comment]
@Eve66666 @AlexeyAB 

The same question. It could be nice if we can save F1,mAP, Ave Loss and IOU during training process. 

I stopped training and when I restarted  the training chart had lost information of last steps

![YOLOv3-Restarted](https://user-images.githubusercontent.com/44882423/81411276-27b68a00-9110-11ea-8d36-20672ff8d1f7.png)

**Comment by AlexeyAB** [comment]
@mfaramarzi Just resume training with `-chart chart.png` at the end of training command.

**Comment by mfaramarzi** [comment]
> @mfaramarzi Just resume training with `-chart chart.png` at the end of training command.

Probably I was supposed to used this command before restarting the training process. Seems the previous "chart.png" is replaced with the above one !

**Comment by mfaramarzi** [comment]
> @mfaramarzi Just resume training with `-chart chart.png` at the end of training command.

Hello Alex,
Why loss drops down when I resume the training?
Thanks for all supports.

![chart](https://user-images.githubusercontent.com/44882423/81689451-4d9da080-9428-11ea-930f-29790f878284.png)

**Comment by AlexeyAB** [comment]
May be you changed something

**Comment by mfaramarzi** [comment]
I stoped training at around 6400 iterations as below:

![chart 2](https://user-images.githubusercontent.com/44882423/81696698-454a6300-9432-11ea-98a2-e21b47518a53.png)

then I used the following command to resume it at 6000 iterations (to have some overlap):

./darknet detector train cfg/damage.data cfg/yolov3-obj.cfg backup/yolov3-obj_6000.weights -chart chart.png

but on the chart new points start appearing at the place it was paused (6400) and not 6000, and loss values are lower than it should be (0.3 instead of 0.6), however the log information make sense as you can see below:

![1](https://user-images.githubusercontent.com/44882423/81697017-ad994480-9432-11ea-8e62-50fbe3b0e2b0.png)

the bottom line is that after resuming logs and chart do not match and logs make sense but not the chart.

**Comment by mfaramarzi** [comment]
@AlexeyAB 

in addition when I resume at 6000,the iteration number is locked on 6518 and it doesn't change

**Comment by AlexeyAB** [comment]
Usually loss drops when you resume the training due to resetting of internal optimizator values, but usually it is temporary and isn't such significant. So this is normal.
